server/model/monitor.js

```diff
@@ -383,6 +383,8 @@ class Monitor extends BeanModel {
                     bean.msg = "Monitor under maintenance";
                     bean.status = MAINTENANCE;
                 } else if (this.type === "group") {
+                    let failChild = "";
```

Comment:
nit:
please don't hamming code variable names here.
I think `downChildrenName` and `downChildrenCount` are better names

Reply:
I'll go with downChildName and failureCount

Reply:
names are different despite close in meaning => don't think this is good for long-term maintence

Reply:
Your petulant complaints have made contributing to this project far more trouble than it is worth. Good day, sir.

Reply:
Your petulant complaints have made contributing to this project far more trouble than it is worth. Good day, sir.

Reply:
I think we are not on the same page what the job as a maintainer is. I see it as making sure that the project can and will move in the right direction.
This does mean that the primary job is to make sure that the code stays maintainable and readable.

The job is not to blindly accept any contribution.

What might be a source of miscommunication is my usage of `nit:` comments. [These comments can be addressed, but does not have to.](https://onetwobytes.com/2021/11/09/nit-code-review/)